src/models/Preset.ts

```diff
@@ -24,7 +24,8 @@ export enum PropertyType {
 export interface PropertyArgs {
   propertyName: string;
   property: CommonModel;
-  type: PropertyType
+  type: PropertyType;
+  modelPropertyName: string;
```

Comment:
For what you need `modelPropertyName`? As I see in the implementation, it's the same as `propertyName` and also you don't use it anywhere.

Reply:
Good catch, left over from old implementation, removing!